.gitignore

```diff
@@ -5,3 +5,5 @@
 build
 
 .idea
+
+python
```

Comment:
just out of curiosity when we are getting this kind of file(?)

Reply:
I use it for azure client